model/src/main/kotlin/licenses/DefaultLicenseInfoProvider.kt

```diff
@@ -82,7 +83,19 @@ class DefaultLicenseInfoProvider(
     private fun createDetectedLicenseInfo(id: Identifier): DetectedLicenseInfo {
         val findings = mutableListOf<Findings>()
 
-        ortResult.getScanResultsForId(id).forEach { (provenance, _, summary, _) ->
+        ortResult.getScanResultsForId(id).forEach { scanResult ->
+            var summary = scanResult.summary
+            var provenance = scanResult.provenance
+
+            if (provenance is RepositoryProvenance) {
```

Comment:
Please add a code comment explaining under which circumstances the re-filtering is meaningful.